apps/darts-modernisation/darts-api/demo.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: helm.toolkit.fluxcd.io/v2beta1
+kind: HelmRelease
+metadata:
+  name: darts-api
+  namespace: darts-modernisation
+spec:
+  releaseName: darts-api
+  values:
+    java:
+      ingressHost: darts-api.demo.platform.hmcts.net
+      image: sdshmctspublic.azurecr.io/darts/api:prod-272ebc2-20230322092842 # {"$imagepolicy": "flux-system:darts-api"}
```

Comment:
This is referenced here https://github.com/hmcts/sds-flux-config/blob/c0b984151aecb0525e18ccc3ce8d60e0db98e69f/apps/darts-modernisation/darts-api/darts-api.yaml